flask_cors/__init__.py

```diff
@@ -157,6 +161,7 @@ def wrapped_function(*args, **kwargs):
                 resp = make_response(f(*args, **kwargs))
 
             _set_cors_headers(resp, options)
+            logger.debug('Response headers: {}'.format(str(resp.headers)))

```

Comment:
I think we should remove this log, as it will be logging a lot of information not specific to CORS, and our specific information is already logged in the set_cors_headers call.